tests/core_tests/test_context.py

```diff
@@ -504,7 +506,7 @@ def test_task_from_sg(self, get_current_user):
         get_current_user.return_value = self.current_user
         
         # add additional field value to task
-        add_value = {"name":"additional", "id": 3, "type": "add_type"}
+        add_value = self.project
```

Comment:
Bad test data. Using non-existing entity type for no good reason. More on why this change was introduced bellow.